test/index.ejs

```diff
@@ -194,20 +200,21 @@
 
     <script src='vendor/jquery.js'></script>
     <script src='vendor/faker.js'></script>
-    <script src='vendor/jquery.formatdatetime.js'></script>
+    <script src='vendor/jquery.formatDateTime.js'></script>
     <script src='vendor/jquery.cookie.js'></script>
     <script src='javascripts/peteshow.js'></script>
-
     <script>
-      Peteshow.init({
-        cookies: true,
-        rules: {
-          "input[name*=middle_name]": Peteshow.random.name(),
-          "input[name*=custom_name]": 'Custom Name'
-        },
-        reuse: { 'input[name*=first_name]': '/tests' },
-        ignore : ['input[name=last_name]', '#qunit-modulefilter']
-      })
+      if ('<%= env %>' != 'test') {
```

Comment:
Ah, I get it now. I would not have guessed that `test/test-server.coffee` and `test/index.ejs` made sense in any context other than a test run, which is why I was confused by all the environment-checking in those files.